<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qq_main_menu</class>
 <widget class="QWidget" name="qq_main_menu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>qq_main_menu</string>
  </property>
  <widget class="QStackedWidget" name="main_stacked_widget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>440</width>
     <height>330</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image:url(:/new/prefix1/res/BackGround2.png)</string>
   </property>
   <property name="currentIndex">
    <number>3</number>
   </property>
   <widget class="QWidget" name="page_1">
    <widget class="QLineEdit" name="LE_login">
     <property name="geometry">
      <rect>
       <x>125</x>
       <y>150</y>
       <width>220</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>220</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>220</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>1060483009</string>
     </property>
     <property name="maxLength">
      <number>16</number>
     </property>
    </widget>
    <widget class="QLineEdit" name="LE_passwd">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>125</x>
       <y>190</y>
       <width>220</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>220</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>220</width>
       <height>30</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>1146011.</string>
     </property>
     <property name="maxLength">
      <number>32</number>
     </property>
    </widget>
    <widget class="QLabel" name="Label_login">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>151</y>
       <width>25</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>25</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>25</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image:url(:/new/prefix1/res/login.png)</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="Label_passwd">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>191</y>
       <width>25</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>25</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>25</width>
       <height>30</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image:url(:/new/prefix1/res/lock.png)</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="Label_top">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>451</width>
       <height>121</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image:url(:/new/prefix1/res/login_background.png)</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="Label_head">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>80</y>
       <width>63</width>
       <height>62</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image:url(:/new/prefix1/res/face.png)</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="Button_register">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>305</y>
       <width>75</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image:url(:/new/prefix1/res/BackGround2.png)</string>
     </property>
     <property name="text">
      <string>注册账号</string>
     </property>
    </widget>
    <widget class="QLabel" name="Label_1">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>150</y>
       <width>271</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(0, 255, 255)</string>
     </property>
     <property name="text">
      <string>__________________________________</string>
     </property>
    </widget>
    <widget class="QLabel" name="Label_2">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>190</y>
       <width>271</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(0, 255, 255)</string>
     </property>
     <property name="text">
      <string>__________________________________</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Button_go">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>235</y>
       <width>210</width>
       <height>35</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image:url(:/new/prefix1/res/white_background.png)</string>
     </property>
     <property name="text">
      <string>登录</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Button_exit">
     <property name="geometry">
      <rect>
       <x>415</x>
       <y>5</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Button_ewm">
     <property name="geometry">
      <rect>
       <x>409</x>
       <y>299</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image:url(:/new/prefix1/res/ewm.png)</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="Lable_log">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>-10</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image:url(:/new/prefix1/res/log_0.png)</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="Button_set">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>5</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>#</string>
     </property>
    </widget>
    <widget class="QLabel" name="Label_netstat">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>305</y>
       <width>240</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="Label_login_stat">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>280</y>
       <width>240</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <zorder>Label_2</zorder>
    <zorder>Label_1</zorder>
    <zorder>LE_login</zorder>
    <zorder>LE_passwd</zorder>
    <zorder>Label_login</zorder>
    <zorder>Label_passwd</zorder>
    <zorder>Label_top</zorder>
    <zorder>Label_head</zorder>
    <zorder>Button_register</zorder>
    <zorder>Button_go</zorder>
    <zorder>Button_exit</zorder>
    <zorder>Button_ewm</zorder>
    <zorder>Lable_log</zorder>
    <zorder>Button_set</zorder>
    <zorder>Label_netstat</zorder>
    <zorder>Label_login_stat</zorder>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QLineEdit" name="LE_login2">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>100</y>
       <width>220</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="maxLength">
      <number>16</number>
     </property>
    </widget>
    <widget class="QLineEdit" name="LE_passwd2">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>140</y>
       <width>220</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="maxLength">
      <number>32</number>
     </property>
    </widget>
    <widget class="QLabel" name="Label_passwd2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>140</y>
       <width>60</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>密码：</string>
     </property>
    </widget>
    <widget class="QLabel" name="Label_login2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>100</y>
       <width>60</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>账号：</string>
     </property>
    </widget>
    <widget class="QLabel" name="Label_message">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>30</y>
       <width>140</width>
       <height>30</height>
      </rect>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::PreventContextMenu</enum>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="Button_register_go">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>180</y>
       <width>140</width>
       <height>45</height>
      </rect>
     </property>
     <property name="text">
      <string>注册</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Button_back_login">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>230</y>
       <width>140</width>
       <height>45</height>
      </rect>
     </property>
     <property name="text">
      <string>返回</string>
     </property>
    </widget>
    <widget class="QLabel" name="Label_netstat2">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>300</y>
       <width>240</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="Label_feedback">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>40</y>
       <width>240</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color:rgb(255, 0, 0)</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_3">
    <widget class="QLabel" name="Label_ewm">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>0</y>
       <width>280</width>
       <height>330</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image:url(:/new/prefix1/res/ewm2.png)</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="Button_ewm2">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>300</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image:url(:/new/prefix1/res/ewm.png)</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_4">
    <widget class="QLineEdit" name="LE_IP">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>100</y>
       <width>240</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>118.31.247.104</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="LE_PT">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>140</y>
       <width>240</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>4397</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="Button_CG_ip">
     <property name="geometry">
      <rect>
       <x>145</x>
       <y>190</y>
       <width>150</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/new/prefix1/res/BackGround1.png);</string>
     </property>
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Button_CG_mr">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>120</y>
       <width>60</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/new/prefix1/res/BackGround1.png);</string>
     </property>
     <property name="text">
      <string>默认</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Button_CG_back">
     <property name="geometry">
      <rect>
       <x>145</x>
       <y>250</y>
       <width>150</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/new/prefix1/res/BackGround1.png);</string>
     </property>
     <property name="text">
      <string>返回</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>100</y>
       <width>75</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>17</pointsize>
      </font>
     </property>
     <property name="text">
      <string>IP</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>140</y>
       <width>75</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>PORT</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_5">
    <widget class="QLabel" name="Label_please_wait">
     <property name="geometry">
      <rect>
       <x>145</x>
       <y>85</y>
       <width>150</width>
       <height>150</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="Button_cancel">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>250</y>
       <width>200</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>17</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">
background-image: url(:/new/prefix1/res/BackGround1.png);</string>
     </property>
     <property name="text">
      <string>取消等待</string>
     </property>
    </widget>
    <widget class="QLabel" name="Label_server_go">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>200</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 0, 0, 255), stop:0.166 rgba(255, 255, 0, 255), stop:0.333 rgba(0, 255, 0, 255), stop:0.5 rgba(0, 255, 255, 255), stop:0.666 rgba(0, 0, 255, 255), stop:0.833 rgba(255, 0, 255, 255), stop:1 rgba(255, 0, 0, 255));</string>
     </property>
     <property name="text">
      <string>正在和服务器沟通...</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
